test/channelSpec.js

```diff
@@ -88,9 +86,17 @@ describe('makePaymentChannelPlugin', function () {
       fulfillment: base64url(this.fulfillment)
     }, requestId + 1, [])
 
+    this.mockSocketIndex = 0
     this.mockSocket = new MockSocket()
-    this.plugin.addSocket(this.mockSocket)
+    this.mockSocket
+      .reply(btpPacket.TYPE_MESSAGE, ({ requestId }) => btpPacket.serializeResponse(requestId, []))
```

Comment:
This `Message` should contain the auth data. Add assertions to test that the client sends the auth data as expected.